am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/repositories/AccessManagementRepository.java

```diff
@@ -100,11 +99,11 @@
 
     @SqlUpdate("insert into access_management_audit (access_management_id, resource_id, accessor_id, permissions, accessor_type, service_name, resource_type, resource_name, attribute, relationship, calling_service_name, audit_timestamp, changed_by, action) "
         + "values (:access_management_id, :resourceId, :accessorId, :permissionsAsInt, cast(:accessorType as accessor_type), :serviceName, :resourceType, :resourceName, :attributeAsString, :relationship,"
-        + " :callingServiceName, CURRENT_TIMESTAMP, :changedBy, 'grant' ) ")
-    void grantAccessManagementForAudit(@Bind("access_management_id") long id, @BindBean ExplicitAccessAuditRecord explicitAccessAuditRecord);
+        + " :callingServiceName, now() at time zone 'utc', :changedBy, 'grant' ) ")
+    void grantAccessManagementForAudit(@Bind("access_management_id") long id, @BindBean ExplicitAccessRecord explicitAccessRecord, String callingServiceName, String changedBy);
```

Comment:
can we use `callingServiceName` and `changedBy` from the `ExplicitAccessRecord` object instead of passing them as individual parameters?